src/ng2-datepicker/locales/en-locale.ts

```diff
@@ -0,0 +1,23 @@
+import { ILocale } from "./locale-manager";
+
+export class EnglishLocale implements ILocale{
+    firstWeekdaySunday(): boolean {
+        return true;
+    }
+    langIdentifier(): string {
+        return "en";
+    }
+    daysOfWeek(): string[] {
+        return ["S", "M", "T", "w", "T", "F", "S"];
+    }
+    clearText(): string {
+        return "Clear";
+    }
+    todayText(): string {
+        return "Today";
+    }
+    selectYearText(): string {
+        return "Select Year";
+    }
+
+}
```

Comment:
you just need to add additional empty line otherwise github is complaining about.